Sources/PackageLoading/Module+PkgConfig.swift

```diff
@@ -78,7 +86,31 @@ private extension SystemPackageProvider {
         }
         return false
     }
-    
+
+    func pkgConfigSearchPath() -> AbsolutePath? {
+        switch self {
+        case .Brew(let name):
+            // Homebrew can have multiple versions of the same package. The
+            // user can choose another version than the latest by running
+            // ``brew switch NAME VERSION``, so we shouldn't assume to link
+            // to the latest version. Instead use the version as symlinked
+            // in /usr/local/opt/(NAME)/lib/pkgconfig.
+            let brewPrefix: String
+            if let prefix = getenv("BREW_PREFIX") {
+                brewPrefix = prefix
+            } else if let prefix = try? Utility.popen(["/usr/local/bin/brew", "--prefix"]).chomp() {
```

Comment:
However; does brew need to be available from Xcode? Once the xcodeproj file is generated, I would expect that it doesn't need brew anymore. The libraries have already been resolved and the correct compiler flags already copied into the xcodeproj file. 

A valid reason for making sure brew is available from Xcode would be debugging `swift build` itself, but I'm not sure on how to achieve that.